core/services/directrequest/delegate.go

```diff
@@ -58,30 +73,37 @@ var (
 )
 
 type listener struct {
-	logBroadcaster  log.Broadcaster
-	unsubscribeLogs func()
-	contractAddress gethCommon.Address
-	pipelineRunner  pipeline.Runner
-	db              *gorm.DB
-	jobID           int32
+	logBroadcaster   log.Broadcaster
+	unsubscribeLogs  func()
+	oracle           oracle_wrapper.OracleInterface
+	pipelineRunner   pipeline.Runner
+	db               *gorm.DB
+	pipelineORM      pipeline.ORM
+	jobID            int32
+	onChainJobSpecID common.Hash
 }
 
 // Start complies with job.Service
-func (d listener) Start() error {
-	connected, unsubscribe := d.logBroadcaster.Register(d, log.ListenerOpts{
-		Contract: nil,
-		Logs:     []generated.AbigenLog{},
+func (l listener) Start() error {
```

Comment:
it was like that before, but shouldn't the receiver be a pointer? otherwise `l.unsubscribeLogs = unsubscribe` will not have any effect, no?